ogi/components/OgImage/gallery.vue

```diff
@@ -0,0 +1,64 @@
+<script setup lang="ts">
+import type { CSSProperties } from 'vue';
+
+// inherited attrs can mess up the satori parser
+defineOptions({
+  inheritAttrs: false,
+});
+
+const props = defineProps<{
+  title: string;
+  image: string;
+  usd: string;
+  price: string;
+  network: string;
+}>();
+
+const cover: ComputedRef<CSSProperties> = computed(() => {
+  return {
+    objectFit: 'cover',
+    objectPosition: 'center',
+  };
+});
+
+const parseUsd = computed(() =>
+  parseFloat(props.usd) ? `$${props.usd}` : '--',
+);
+</script>
+
+<template>
+  <img :src="image" :alt="title" :style="cover" class="h-full w-full" />
+
+  <div
+    class="flex flex-col justify-end h-full w-full bg-slate-900/85 text-white p-20 text-2xl font-bold absolute inset-0"
```

Comment:
let's unify padding (https://github.com/kodadot/workers/pull/198/files) your solution might be better